packages/theme/stories/foundations/size.stories.mdx

```diff
@@ -11,16 +11,22 @@ All size values are based on 4px, `--salt-size-basis-unit`. If you need to chang
 ## Tokens
 
 <DocGrid>
+  <SpacingBlock spacingVar="--salt-size-accent" />
   <SpacingBlock spacingVar="--salt-size-basis-unit" />
   <SpacingBlock spacingVar="--salt-size-border" />
-  <SpacingBlock spacingVar="--salt-size-accent" />
+  <SpacingBlock spacingVar="--salt-size-brandBar" /> **deprecated**: use --salt-size-accent
```

Comment:
I think we need to think how to display this ones, it is currently breaking the grid